presentation/src/main/java/me/androidbox/presentation/agenda/screen/AgendaScreenEvent.kt

```diff
@@ -0,0 +1,7 @@
+package me.androidbox.presentation.agenda.screen
+
+import java.time.LocalDate
+
+sealed interface AgendaScreenEvent {
+    data class OnDateChanged(val date: LocalDate) : AgendaScreenEvent
+}
```

Comment:
@philipplackner @MKiperszmid Is it worth having a sealed interface for a single event? It could be possible we start with one then scale up to other events as the project grows. So I think it will be worth keeping it.

Reply:
This screen will have more than just 1 event

Stuff like:
OnDayOfWeekChange
OnItemClick
OnNewItem

etc

So I would keep it, yes

Reply:
Yeah, even if you'd only have one event, it's likely a screen gets more in future with changing requirements. It also adds some sort of consistency.